src/main/java/bio/terra/cli/context/WorkspaceContext.java

```diff
@@ -99,6 +99,32 @@ public void deleteWorkspace() {
     writeToFile();
   }
 
+  /**
+   * Getter for the Terra workspace display name. Returns empty string if the display name is not
+   * defined.
+   *
+   * @return the Terra workspace display name
+   */
+  @JsonIgnore
+  public String getWorkspaceDisplayName() {
+    return terraWorkspaceModel == null || terraWorkspaceModel.getDisplayName() == null
+        ? ""
```

Comment:
could this just return `Optional<String>`? I don't like passing around empty strings because they frequently have to be filtered out later.